.goreleaser.yml

```diff
@@ -1,8 +1,5 @@
 archives:
-  - files:
-      # Only include built binary in archive
-      - 'none*'
```

Comment:
I'd recommend keeping this configuration, unless there is a specific reason to ship additional files inside the archives (which Terraform CLI will unpack into plugin directories)